input/sources/tev-000147.yaml

```diff
@@ -26,6 +26,8 @@ pos:
 reference_ids:
```

Comment:
On the hess source catalog webpage there is also mentioned AICP 848 (2006) 570 as reference. Do we want to add this reference?